charts/tvheadend/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+appVersion: 4.2
```

Comment:
Actually looking at LSIO image tags for this image, I would roll with `appVersion: latest` here.

Reply:
No probs! Done